src/System.Runtime.Extensions/src/Resources/Strings.resx

```diff
@@ -1,5 +1,64 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <root>
+  <!-- 
```

Comment:
Note that for some reasons VS 2017 Community Edition has added spaces in some places when I added a new error message.